src/components/productoverview/PageProductOverview.vue

```diff
@@ -74,7 +75,7 @@
             </div>
           </form>
 
-          <ProductList :categorySlug="categorySlug" />
+          <ProductList :categorySlug="categorySlug" :sort="sort"/>
```

Comment:
Now that I see the whole implementation, I think it is best to have all filter/sort/pagination/etc., all inside `ProductList` component. This way ProductList can orchestrate all the components that influence the query. This also facilitates a lot any modification to developers.

I'm working on an approach that will make it really meaningful, but meanwhile you can start by putting all the `<form>` above inside ProductList.